src/test/java/systemtests/AddressBookSystemTest.java

```diff
@@ -215,14 +215,15 @@ protected void assertSelectedCardDeselected() {
      */
     protected void assertSelectedCardChanged(Index expectedSelectedCardIndex) {
```

Comment:
This test is a bit dubious now since the actual loaded URL is going to be quite different from the link depending on the circumstances. We may have to remove this test and leave it for manual testing

Reply:
Yes, I've removed the assertion on the expected and loaded URL, but left the card-related assertions as they are.